### DISCUSSION TITLE
How Does Faiss store its data structure on disk?

### DISCUSSION THREAD

**Discussion by TalalAhmed311** [facebookresearch/faiss#3240] [Q&A]
So, I want to understand the process for on-disk. Most of the databases create graph data structures for optimizing but I have a confusion about when its time to persist data then how does this data structure get stored on disk?

**Comment by xrd** [comment]
This might be useful:

https://github.com/facebookresearch/faiss/wiki/Indexes-that-do-not-fit-in-RAM

Here is sample code:

https://github.com/facebookresearch/faiss/blob/main/demos/demo_client_server_ivf.py
